packages/react-ui-ag/src/Filters/RadioGroupDropdown.js

```diff
@@ -93,10 +93,10 @@ export default class RadioGroupDropdown extends Component {
 }
 
 const RadioGroupDropdownAnchorTextComponent = props => (
-  <React.Fragment>
+  <div>
```

Comment:
Jest doesn't play nice when testing React.Fragments within the newest version ☹️ . Hopefully we can switch this back later.